tests/ui/err-01.stderr

```diff
@@ -1,12 +1,16 @@
+warning: `$DIR/target/tests/fatality/.cargo/config` is deprecated in favor of `config.toml`
```

Comment:
Any idea where this is coming from? It definitely shouldn't be in the output but I can't find where it is generated.

Reply:
It was from trybuild. Bumped it too to fix this.